content/git/concepts/branch/branch.md

```diff
@@ -36,6 +36,18 @@ git branch -d <branch-name>
 
 As long as the branch has no uncommitted code, this command will remove the branch entirely. It will no longer be available to check out, and the changed code will be unrecoverable.
 
+Sometimes Git will refuse to delete a branch because it has changes that have not been merged into the upstream or remote version of that branch. In these cases, Git can be _forced_ to delete the branch by changing the flag to a capital "D" or adding the `--force` flag like so:
+
+```shell
+# Option 1:
+git branch -D <branch-name>
+
+# Option 2:
+git branch -d --force <branch-name>
+```
+
+By performing a force delete, the user is telling Git that they understand all changes on that branch will be lost forever, and they want to perform the delete anyway.
+
 ## Moving Between Branches
 
 The basic syntax for moving between branches in Git is this:
```

Comment:
A couple a thoughts to note about lines #53-57:

* Let's change the sentence in line #53 to exclude "basic" from it.
* On lines #55-57, let's change `<branch-name>` to `branch-name` to make the pseudocode syntactically compliant w/ Git (running with `<`,`>` yields an error).
* * Since this section talks about moving between branches, should we include information about [the `git switch` command](ff3e4e1bce8bcc8068623639aa5f0272e6e6c49f)?

Reply:
Sure, no problem.

Regarding `git switch`, that might be better added as a separate entry in a new PR, if at all. `git switch` and `git restore` are still considered experimental and I don't personally know anyone that uses them, but they have been around for a while now, so maybe it makes sense to add them.  If nothing else, it may help clear up some confusion about how to use `checkout`.

Reply:
That sounds good! We should consider those in the near future but not for this PR. 😄